base/body_test.go

```diff
@@ -59,7 +59,7 @@ func TestDatasetBodyFile(t *testing.T) {
 		{&dataset.Dataset{Structure: &dataset.Structure{Format: "csv"}, BodyBytes: []byte("a,b,c\n1,2,3")}, "body.csv", 11, ""},
 
 		// urlz
-		{&dataset.Dataset{BodyPath: "http://"}, "", 0, "fetching body url: Get \"http:\": http: no Host in request URL"},
+		{&dataset.Dataset{BodyPath: "http://"}, "", 0, "fetching body url: Get http:: http: no Host in request URL"},
```

Comment:
After a slack chat, we've come to realize this change is actually a question of what versions of go we'd like to support. If it's 1.14 this change needs to be reverted, if it's 1.13, we need to generalize this test to accommodate both versions.

Our docs say 1.13 is minimum, we voted to generalize.